### ISSUE TITLE
[Bug] `distrobox assemble rm` hangs when replacing container with custom home

### ISSUE THREAD

**Issue by awesomekyle** [89luca89/distrobox#855]
Please, before opening a bug:

- make sure you've read the documentation.
- Ensure there isn't already an open issue about this.
- Ensure there isn't already a closed/resolved issue about this.

**Describe the bug**
When using `assemble` to replace or remove a container with a custom home, the command appears to hangs because the `distrobox rm` command is waiting for input.

**To Reproduce**
1. Create manifest with contents:
```
[testdb]
image=quay.io/toolbx-images/alpine-toolbox:latest
replace=true
home=$HOME/test-home
```
2. `distrobox assemble create --file path-to.ini` to create initial container
3. `distrobox assemble create --file path-to.ini` to replace and recreate container
4. See output hang at ` - Deleting testdb... `. It's waiting for input at `distrobox-rm#257` ("Do you want to remove custom home of container...")

**Expected behavior**
I expect the container to be removed/replaced without destroying the custom home, or to see the prompt asking what to do.

**Logs**
[Log when replacing](https://github.com/89luca89/distrobox/files/12034417/replace.log)

**Desktop (please complete the following information):**
 - Are you using podman or docker?
 - Which version or podman or docker?
 - Which version of distrobox?

**Additional context**
`distrobox assemble rm`, or `distrobox assemble create` after it's already created, tries to execute `/usr/local/bin/distrobox rm -f testdb`. When executed manually, this is the output:

```
❯ /usr/local/bin/distrobox rm -f testdb              
Do you want to remove custom home of container testdb (/tmp/test-home)? [y/N]:
```
Because `distrobox-rm` defaults to `N` for that question, hitting enter will allow it to continue without deleting the user's custom home. Either the prompt should show up so the user knows that it's not hung, and/or it should be possible to give the answer in the ini.

**Comment by RashiqAzhan** [comment]
My one gets stuck even without a custom home.
distrobox.ini
```ini
[latex-archlinux]
image=quay.io/toolbx-images/archlinux-toolbox:latest
pull=true
replace=true
start_now=true
additional_packages=tectonic
volume=/nvme_storage:/nvme_storage
```

Last terminal output with `distrobox assemble create -v`
```
++ image=quay.io/toolbx-images/archlinux-toolbox:latest
++ pull=1
++ replace=1
++ start_now=1
++ additional_packages=tectonic
++ volume=/nvme_storage:/nvme_storage
+ rm -f /tmp/tmp.LvCw0PpVCl
+ '[' 0 -ne 0 ']'
+ '[' 1 -ne 0 ']'
+ printf ' - Deleting %s... \n' latex-archlinux
 - Deleting latex-archlinux... 
+ '[' 0 -eq 0 ']'
+ /usr/bin/distrobox rm -f latex-archlinux
```

**Comment by 89luca89** [comment]
This is now solved by e18e01844e71d38af9b5cd67ac0e49d81cd735d2
